Clarify "new information about the user’s intent" definition

anssiko opened the pull request:
Add an informative definition for implicit signals.

Fix downstream issue https://github.com/w3c/sensors/issues/211

PTAL @marcoscaceres @mounirlamouri @jyasskin


<!--
    This comment and the below content is programatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
[Preview](https://s3.amazonaws.com/pr-preview/anssiko/permissions/implicit-signals.html) | [Diff](https://s3.amazonaws.com/pr-preview/w3c/permissions/120fe18...anssiko:8fbabc5.html)

anssiko commented:
@marcoscaceres, can I again bribe you with :beers: at TPAC?

jyasskin commented on index.bs:
```diff
@@ -115,7 +118,16 @@ spec: webidl
+      or <a>implicit signals</a> this specification hasn't anticipated.
+      <div class="note">
+        The <dfn>implicit signals</dfn> could be, for example, the
+        <a>install</a> status of a web application, frequency and recency of
```
"... application *or* frequency and recency ...", I think.

Or to reword the whole sentence:
> For example, a UA could use the user's frequency and recency of visits to the application, or the fact they've [=installed=] it, as <dfn>implicit signals</dfn>, since a user that ...

anssiko replied:
Fixed.

jyasskin commented on index.bs:
```diff
@@ -115,7 +118,16 @@ spec: webidl
+        The <dfn>implicit signals</dfn> could be, for example, the
+        <a>install</a> status of a web application, frequency and recency of
+        visits. A user that has installed a web application and used it
+        frequently and recently is more likely to have established a trust
```
s/to have established a trust relationship with the application provider/trust it/?

anssiko replied:
You're proposal is more concise, I like it. Fixed.

jyasskin commented on index.bs:
```diff
@@ -115,7 +118,16 @@ spec: webidl
+        visits. A user that has installed a web application and used it
+        frequently and recently is more likely to have established a trust
+        relationship with the application provider. Implementations are
+        advised to exercise caution when implementing strategies that rely on
```
s/implementing strategies that rely/relying/?

anssiko replied:
Ditto. Fixed.

jyasskin commented on index.bs:
```diff
@@ -115,7 +118,16 @@ spec: webidl
+        visits. A user that has installed a web application and used it
+        frequently and recently is more likely to have established a trust
+        relationship with the application provider. Implementations are
+        advised to exercise caution when implementing strategies that rely on
```
s/are advised to exercise caution/should be cautious/?

anssiko replied:
I want to avoid using RFC 2119 terms in information notes, so I'll will leave this as is.

jyasskin replied:
SG.

jyasskin commented on index.bs:
```diff
@@ -41,6 +41,9 @@ spec: mediacapture-main; urlPrefix: https://w3c.github.io/mediacapture-main/#
         text: generic sensor permission revocation algorithm; url: generic-sensor-permission-revocation-algorithm
+spec: manifest; urlPrefix: https://w3c.github.io/manifest/#
+    type: dfn
+        text: install; url: dfn-install
```
Once https://github.com/tabatkins/bikeshed/issues/1088 is fixed, you should be able to remove this.

anssiko replied:
I will leave this in while waiting for that upstream patch to land, ok?

jyasskin replied:
Yep.

anssiko commented:
@jyasskin, thanks a lot for your review! I've addressed your comments in https://github.com/w3c/permissions/pull/154/commits/8fbabc500a9889021686d0680a28c3e221a93644

Please let me know if you'd like me to squash the commits together.

anssiko commented:
Thanks!
